docker/Dockerfile.omp_offload

```diff
@@ -0,0 +1,110 @@
+RUN apt-get install -y --no-install-recommends libnuma-dev
+RUN apt-get install -y --no-install-recommends gcc-multilib
+RUN apt-get install -y --no-install-recommends g++-multilib
+RUN apt-get install -y --no-install-recommends wget
+# latest cmake
+RUN apt-get install -y --no-install-recommends software-properties-common
+RUN apt-get install -y --no-install-recommends lsb-release
+RUN wget -q -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
+RUN apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
+RUN apt-get update
+RUN apt-get -y --no-install-recommends install cmake
+
+## INSTALL CLANG 13
+
+RUN wget -q https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-13.0.0.tar.gz
+RUN tar -xf llvmorg-13.0.0.tar.gz
+WORKDIR /llvm-project-llvmorg-13.0.0
+RUN mkdir build
+WORKDIR /llvm-project-llvmorg-13.0.0/build
+RUN cmake ../llvm/ -DCMAKE_BUILD_TYPE=Release -DLLVM_TARGETS_TO_BUILD="X86;NVPTX" -DLIBOMPTARGET_NVPTX_COMPUTE_CAPABILITIES=all -DLLVM_ENABLE_PROJECTS="clang" -DLLVM_ENABLE_RUNTIMES="openmp" -DCMAKE_INSTALL_PREFIX=/llvm13
+RUN make -j $(( $(lscpu | awk '/^Socket\(s\)/{ print $2 }') * $(lscpu | awk '/^Core\(s\) per socket/{ print $4 }') ))
+RUN make install
+WORKDIR /
+RUN rm -rf llvmorg-13.0.0.tar.gz llvm-project-llvmorg-13.0.0
+ENV PATH /llvm13/bin:${PATH}
+ENV LD_LIBRARY_PATH /llvm13/lib:${LD_LIBRARY_PATH}
+
+## INSTALL PYTHON
+
+RUN apt-get install -y --no-install-recommends python3-dev
+RUN apt-get install -y --no-install-recommends python3-venv
+
+## INSTALL DEVITO
+
+ADD ./requirements.txt /app/requirements.txt
+ADD ./requirements-optional.txt /app/requirements-optional.txt
+# ADD ./requirements-mpi.txt /app/requirements-mpi.txt
+
+RUN python3 -m venv /venv
+RUN /venv/bin/pip install --no-cache-dir --upgrade pip
+RUN /venv/bin/pip install --no-cache-dir jupyter
+RUN /venv/bin/pip install --no-cache-dir wheel
+RUN /venv/bin/pip install --no-cache-dir -r /app/requirements.txt
+RUN /venv/bin/pip install --no-cache-dir -r /app/requirements-optional.txt
+# RUN /venv/bin/pip install --no-cache-dir -r /app/requirements-mpi.txt
+RUN rm -rf ~/.cache/pip
+
+ADD ./devito /app/devito
+ADD ./tests /app/tests
+ADD ./examples /app/examples
+ADD ./benchmarks /app/benchmarks
+COPY setup.cfg /app/
+ADD docker/entrypoint.sh /docker-entrypoint.sh
+
+RUN chmod +x /docker-entrypoint.sh
+RUN apt-get install -y --no-install-recommends nano
+## CREATE APP USER
+
+# Set the home directory to our app user's home.
+ENV HOME=/app
+ENV APP_HOME=/app
+
+# Create the home directory for the new app user.
+# Create an app user so our program doesn't run as root.
+# Chown all the files to the app user.
+RUN mkdir -p /app
+RUN groupadd -r app
+RUN useradd -r -g app -d /app -s /sbin/nologin -c "Docker image user" app
+RUN chown -R app:app $APP_HOME
+RUN chown -R app:app /docker-entrypoint.sh
+RUN chown -R app:app /venv
```

Comment:
I know it's from the nvidia docker image but this really seems like overkill.

Reply:
I'm not sure I understood. Are you suggesting removing the creation of a non-admin user?